packages/open-next/src/build/patch/patches/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./patchEnvVar.js";
```

Comment:
nit: I think `export *` in a barrel is kind of an anti-pattern, could we change to named exports?